xm-commons-scheduler/src/main/java/com/icthh/xm/commons/scheduler/service/SchedulerEventHandlerFacade.java

```diff
@@ -47,6 +51,10 @@ public void onMessage(String message, String tenant, TopicConfig topicConfig) {
         }
     }
 
+    private ObjectMapper initObjectMapper() {
+        return new ObjectMapper().registerModule(new JavaTimeModule());
```

Comment:
if i'm not mistaken ObjectMapper in commons i always defined via new and could have it's own configuration. I suppose that it was designed to avoid bean definition conflicts as one reason. Do I understand right that you want to use mapper from spring context here?